src/tls/ca.h

```diff
@@ -18,11 +17,11 @@ namespace tls
     Unique_X509 ca;
 
   public:
-    CA(CBuffer ca_ = nullb)
+    CA(std::span<const uint8_t> ca_ = {})
```

Comment:
Since all the callers have this as a c-string or similar, can we move the awkward cast inside this constructor (eg, make this take a `std::span<const char> ca_`, or `char const* ca_, size_t ca_len`)?

Reply:
There's a test one that doesn't, easily changed, but also the config, so it's not quite as nice as that.